src/taco-remote/lib/buildManager.ts

```diff
@@ -150,6 +151,7 @@ class BuildManager {
             params.buildNumber = buildNumber;
             params.options = options;
             params.logLevel = logLevel;
+            params.target = req.query.target;
```

Comment:
Isn't this a no-op since `params = req.query`, so `params.target` is identical to `req.query.target` already?